common/artifacts/000_wc_artifacts.txt

```diff
@@ -371,7 +371,6 @@ runeblade_blood_3 = {
 	picture = "GFX_runeblade_blood"
 	slot = weapon
 }
-
```

Comment:
I don't see any changes in artifact files, besides deleting whitespaces
And because of this, github gone crazy
<details>

![Безымянный2](https://user-images.githubusercontent.com/46576860/63954147-4e092d80-ca8b-11e9-83c8-3059285997fb.png)

</details>
It deleting these whitespaces is really necessary?

Reply:
I thought the same way about the whitespaces. I thought, since they served no (obvious) purpose, I could delete them.